app/_locales/en/messages.json

```diff
@@ -678,6 +678,9 @@
   "close": {
     "message": "Close"
   },
+  "coingecko": {
+    "message": "Coingecko"
```

Comment:
```suggestion
    "message": "CoinGecko"
```

I think we'll want to update `Coingecko` -> `CoinGecko`
<img width="761" alt="Screen Shot 2022-12-22 at 03 24 18" src="https://user-images.githubusercontent.com/20778143/208996456-d4d7b767-f5a8-4232-9514-a862e36c8c58.png">